src_assets/common/assets/web/config.html

```diff
@@ -358,7 +358,7 @@ <h2 class="accordion-header">
         </div>
 
         <!-- Native pen/touch support -->
-        <div class="mb-3" v-if="config.mouse === 'enabled' && platform === 'windows'">
+        <div class="mb-3" v-if="config.mouse === 'enabled'">
```

Comment:
Can we merge https://github.com/LizardByte/Sunshine/pull/2002 and https://github.com/LizardByte/Sunshine/pull/2000 first?

Reply:
Yeah, that's fine with me, though I don't think this will conflict with your PRs anyway.

Reply:
Yeah, that's fine with me, though I don't think this will conflict with your PRs anyway.

Reply:
You're right and this can be merged prior. Since it's just one line it won't be that difficult to rebase mine if I had to.

For some reason I was thinking that I had added ids to every option in #2000, but looks like most already had it... and this one has it below.

So this is good to merge IMO, if you're happy with it.